### ISSUE TITLE
flux mini run fails on Spack built flux-sched

### ISSUE THREAD

**Issue by wkharold** [flux-framework/flux-sched#931]
Built flux-sched on CentOS 7 with Spack 0.17.1

running ```flux mini run -n4 --label-io hostname``` failed with

    2022-04-13T16:46:43.100676Z job-ingest.err[0]: validator[0]: exited with rc=1
    flux-mini: ERROR: [Errno 71] Unrecoverable error: validator unexpectedly exited

It failed after a Slurm `salloc` and after `flux start --test-size=4`

**Comment by grondo** [comment]
can you paste the output of `flux dmesg` after a failure? That should show the output of the failing validator (my guess is some Python issue)

**Comment by wkharold** [comment]
sure. here's a [gist of the output](https://gist.github.com/wkharold/68aa609a92da152667f7af0af6cd772b)

TL;DR it appears to be complaining about not finding the 'jobspec' validator plugin

I'm pretty sure this worked without a problem when I manually installed on Rocky Linux. This installation was automated, using spack, as part of a Slurm cluster install.
